.github/workflows/continuous-integration.yml

```diff
@@ -54,17 +54,17 @@ jobs:
     # - name: Pull the Docker image from DockerHub
     #   run: docker pull sjain309/todo-app-prod
 
-    - name: Tag the Docker image for Heroku
-      run: docker tag sjain309/todo-app-prod registry.heroku.com/heroku-app-devops/web
+    # - name: Tag the Docker image for Heroku
+    #   run: docker tag sjain309/todo-app-prod registry.heroku.com/heroku-app-devops/web
 
-    - name: Login to Heroku container
-      run: heroku container:login
+    # - name: Login to Heroku container
+    #   run: heroku container:login
 
-    - name: Push it to Heroku registry
-      run: docker push registry.heroku.com/heroku-app-devops/web
+    # - name: Push it to Heroku registry
+    #   run: docker push registry.heroku.com/heroku-app-devops/web
 
-    - name: Release Heroku container
-      run: heroku container:release web --app heroku-app-devops
+    # - name: Release Heroku container
+    #   run: heroku container:release web --app heroku-app-devops
```

Comment:
You should always delete code rather than comment it out. It will live on in the git history if you need to restore it.